myhpi/core/markdown/extensions.py

```diff
@@ -160,6 +160,21 @@ def url(self, id):
     def default_pattern():
         return r"\[(?P<title>[^\[]+)\]\(page:(?P<id>\d+)\)"
 
+class ExternalLinkPattern(LinkInlineProcessor):
+    def handleMatch(self, m, data=None):
+        el = util.etree.Element("a")
+        try:
+            el.set("href", m.group("url"))
```

Comment:
is it necessary to re-set the href here? maybe you should check the priority of markdowns built-in link parser and whether you want to replace or supplement it